tests/unit/pomodoro_test.sh

```diff
@@ -380,4 +387,54 @@ function test_show_tags()
   assert_equals_helper 'Wrong output' "$LINENO" "$expected" "$output"
 }
 
+function test_fetch_last_pomodoro_session()
```

Comment:
Create one test that validates the condition that we don't have any previous session, and the user tries to use this option. Use a new test function for that.